backtracking/n_queens_math.py

```diff
@@ -0,0 +1,140 @@
+"""
```

Comment:
```suggestion
r"""
```
Use a `raw string` to use a backslash.